Bottom: 3bdaadb59a01276b726332bd9c4ccf43ef78c08f
Top:    0fbd482b4ee9145e92516bfd2b006a040a50fb35
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-30 15:49:13 +0800

Refresh of patch

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index cfb11c9e..0d99b78 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -535,11 +535,12 @@ static void handle_tx(struct vhost_net *net)
 		.msg_flags = MSG_DONTWAIT,
 	};
 	size_t len, total_len = 0;
+	int err;
 	struct socket *sock;
 	struct vhost_net_ubuf_ref *uninitialized_var(ubufs);
 	struct vhost_used_elem used;
 	bool zcopy, zcopy_used;
-	int err, sent_pkts = 0;
+	int sent_pkts = 0;
 
 	mutex_lock(&vq->mutex);
 	sock = vq->private_data;
